OneCryptoPkg/Include/Library/HmacLib.h

```diff
@@ -0,0 +1,292 @@
+/** @file
+  HMAC Library provides HMAC cryptographic services
+
+  Copyright (C) Microsoft Corporation. All rights reserved.
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+
+**/
+
+// ------------------------------------------------------------------------------
+// AUTOGENERATED BY CreateCryptoProtocol.py
```

Comment:
Update all comments indicating that this was auto generated to say

MANUALLY UPDATED
DATE: <date>
VERSION: <version>